src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions.txt

```diff
@@ -49,7 +49,7 @@ Table()
 Table()
 
 // Filter blank table
->> Filter(Blank(), Blank())
+>> Filter(If(1<0,[1,2,3]), Blank())
```

Comment:
What is  `Filter(Blank(), Blank())` ?  If that's now a compiler error, we can still include the test and enforce that.